src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -66,6 +66,8 @@ ArcGisMapServerItemViewModel.prototype._enableInCesium = function() {
 
     var scene = this.application.cesium.scene;
 
+    this.legendUrl = this.url + '/legend';
```

Comment:
It's not there.  It's been removed in all instances.  It was a hack then and it hasn't improved with age.